beacon-chain/rpc/beacon/config.go

```diff
@@ -0,0 +1,15 @@
+package beacon
+
+import (
+	"context"
+
+	ptypes "github.com/gogo/protobuf/types"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// GetBeaconConfig returns the configuration of the beacon chain as understood by this node.
+func (bs *Server) GetBeaconConfig(ctx context.Context, _ *ptypes.Empty) (*ethpb.BeaconConfig, error) {
```

Comment:
Was this part of the validator status changes in ethereumapis?

Reply:
No, but because that PR came in after the one that added GetBeaconConfig() the commit of ethereumapis that we pull in includes this.  #4595 will fill this function in (at least, that's what it says it will do; at current the PR isn't populated with the required code).